registry/templates/donor/award_prep.html

```diff
@@ -46,6 +46,7 @@ <h1>Přehled dárců k ocenění: {{ medal.title }}</h1>
 
 <input type="submit" class="btn btn btn-success" value="Udělit medaili označeným dárcům">
 </form>
+<input type="checkbox" onClick="toggle(this)"> Vybrat vše
```

Comment:
Ten bych tam možná nedával. Je to trošku vzdálené od výběru ve sloupci, ale můžeme to probrat.